practices/practice-one/src/styles/sections/header.css

```diff
@@ -0,0 +1,15 @@
+/* add to late */
+.header-wrapper {
```

Comment:
I have shared with you about CSS selector via slack, please check then update

Reply:
I have done that.